### DISCUSSION TITLE
diary/2017/12/08/diary/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#153] [Comments]
# diary/2017/12/08/diary/

개발 블로그 시작 기존엔 네이버 블로그를 하고 있었는데, 네이버 블로그는 상품 혹은 경험 관련 내용을 위주로 올려서(예전에 마케팅, 광고를 했던 경험 기반) 개발 내용을 추가하기 쉽지 않았다. 게다가 구글 검색에 결과가 잘 나오지 않아 Github 블로그를 만들기 시작! 목표는 Github TIL 폴더에 있는 내용들을 다 텍스트로 정리하는 것! 사실 깃허브에...

https://zzsza.github.io/diary/2017/12/08/diary/

**Comment by partrita** [comment]
블로그가 굉장히 깔끔하네요ㅎ
